EventGenerator/fcl/prolog.fcl

```diff
@@ -108,13 +121,6 @@ StoppedParticleReactionGun: {
 }
 
 
-StoppedParticleReactionGunN: {
-    @table::StoppedParticleReactionGun
-    module_type    : StoppedParticleReactionGunN
-    stashSize      : @nil
-}
```

Comment:
I presume that StoppedParticleReactionGunN was removed in an earlier commit and this is just left over cleanup?

Reply:
Yes, that is correct.